test/quadquizaminos_web/live/power_up_test.exs

```diff
@@ -12,6 +12,15 @@ defmodule QuadquizaminosWeb.PowerUpTest do
     [user: user, conn: conn]
   end
 
+  test "powerup hover display name of respective powerups", %{conn: conn} do
+    {view, _html} = pause_game(conn)
+
+    html = render_click(view, "keydown", %{"key" => "p"})
```

Comment:
Once we remove pressing `p` which is currently used for debugging, will this test fail?

Reply:
yeah it will, I have refactored the test